core/src/test/scala/ai/h2o/sparkling/ConfigurationPropertiesTestSuite.scala

```diff
@@ -35,42 +34,6 @@ abstract class ConfigurationPropertiesTestSuite
   with SparkTestContext {
 
   @transient var hc: H2OContext = _
-
-  override def afterEach(): Unit = {
-    // The method H2O.exit calls System.exit which confuses Gradle and marks the build
-    // as successful even though some tests failed.
-    // We can solve this by using security manager which forbids System.exit call.
-    // It is safe to use as all the methods closing H2O cloud and stopping operations have been
-    // already called and we just need to ensure that JVM with the client/driver doesn't call the System.exit method
-    try {
-      val securityManager = new NoExitCheckSecurityManager
-      System.setSecurityManager(securityManager)
-      if (hc != null) {
-        hc.stop()
```

Comment:
Don't we still need to call this line?

Reply:
Nope :) It is called via normal execution path -> `stop` method during spark shutdown hook